docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -0,0 +1,884 @@
+    "\n",
+    "\n",
+    "## Example Network\n",
+    "\n",
+    "We use a simple network with 3 nodes, 1 source, 3 lines, and 2 loads (1 symmetric load, 1 asymmetric load). As shown below:\n",
+    "\n",
+    "```\n",
+    " -----------------------line_8---------------\n",
+    " |                                          |\n",
+    "node_1 ---line_3--- node_2 ----line_5---- node_6\n",
+    " |                    |                     |\n",
+    "source_10          sym_load_4           asym_load_7\n",
+    "```\n",
+    "\n",
+    "The 3 nodes are connected in a triangular way by 3 lines.\n",
+    "\n",
+    "**NOTE: load_7 is asymmetric in this case.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "ae11dc9a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# some basic imports\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from power_grid_model import LoadGenType\n",
+    "from power_grid_model import PowerGridModel, CalculationMethod, CalculationType, MeasuredTerminalType\n",
+    "from power_grid_model import initialize_array"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f983cef7",
+   "metadata": {},
+   "source": [
+    "## Power Flow Calculation\n",
+    "### Input Dataset for State Estimation\n",
+    "\n",
+    "We create input dataset by using the helper function `initialize_array`.\n",
+    "Note the units of all input are standard SI unit without any prefix,\n",
+    "i.e. the unit of voltage is volt (V), not kV.\n",
+    "\n",
+    "Please refer [Components](https://power-grid-model.readthedocs.io/en/stable/user_manual/components.html) for detailed explanation of all component types and their input/output attributes. \n",
+    "\n",
+    "**NOTE: The required attributes of each components can be different for asymmetric calculations.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "6f008736",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# node\n",
+    "node = initialize_array(\"input\", \"node\", 3)\n",
+    "node[\"id\"] = np.array([1, 2, 6])\n",
+    "node[\"u_rated\"] = [10.5e3, 10.5e3, 10.5e3]\n",
+    "\n",
+    "# line\n",
+    "line = initialize_array(\"input\", \"line\", 3)\n",
+    "line[\"id\"] = [3, 5, 8]\n",
+    "line[\"from_node\"] = [1, 2, 1]\n",
+    "line[\"to_node\"] = [2, 6, 6]\n",
+    "line[\"from_status\"] = [1, 1, 1]\n",
+    "line[\"to_status\"] = [1, 1, 1]\n",
+    "line[\"r1\"] = [0.25, 0.25, 0.25]\n",
+    "line[\"x1\"] = [0.2, 0.2, 0.2]\n",
+    "line[\"c1\"] = [10e-6, 10e-6, 10e-6]\n",
+    "line[\"tan1\"] = [0.0, 0.0, 0.0]\n",
+    "line[\"i_n\"] = [1000, 1000, 1000]\n",
+    "line[\"r0\"] = [0.25, 0.25, 0.25]\n",
+    "line[\"x0\"] = [0.2, 0.2, 0.2]\n",
+    "line[\"c0\"] = [10e-6, 10e-6, 10e-6]\n",
+    "line[\"tan0\"] = [0, 0, 0]\n",
```

Comment:
it could be interesting to see what happens when you change these parameters e.g. in one of the batch scenarios